src/core/sent_packet_metadata.h

```diff
@@ -88,12 +88,47 @@ typedef struct QUIC_SEND_PACKET_FLAGS {
     BOOLEAN IsMtuProbe              : 1;
     BOOLEAN KeyPhase                : 1;
     BOOLEAN SuspectedLost           : 1;
+    BOOLEAN IsAppLimited            : 1;
+    BOOLEAN HasLastAckedPacketInfo  : 1;
 #if DEBUG
     BOOLEAN Freed                   : 1;
 #endif
 
 } QUIC_SEND_PACKET_FLAGS;
 
+//
+// Packet info of last acked packet
+//
+typedef struct LAST_ACKED_PACKET_INFO {
+
+    //
+    // Total bytes sent
```

Comment:
If the field is called "Total bytes sent", then adding a code comment that says "total bytes sent" adds zero value and is annoying to read.

I think the only fields in this struct that need comments are AdjustedAckTime and AckTime.

Reply:
Actually, we also need a comment explaining the difference between the TotalBytesSent field here and the one in QUIC_SENT_PACKET_METADATA.